package.json

```diff
@@ -22,7 +22,71 @@
   },
   "license": "MIT",
   "author": "stylelint",
-  "main": "lib/index.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./lib/index.mjs",
+      "require": "./lib/index.cjs"
+    },
+    "./package.json": "./package.json",
+    "./lib/utils/isStandardSyntaxAtRule": {
```

Comment:
Thanks, I missed the [Subpath patterns](https://nodejs.org/api/packages.html#subpath-patterns) feature. It can simplify the settings. Fixed with 027567a07.